source/vibe/core/net.d

```diff
@@ -137,6 +137,12 @@ TCPListener listenTCP(ushort port, TCPConnectionDelegate connection_callback, st
 		sopts |= StreamListenOptions.reusePort;
 	else
 		sopts &= ~StreamListenOptions.reusePort;
+	version(linux) {
```

Comment:
```suggestion
	static if (is(typeof(StreamListenOptions.ipTransparent))) {
```

That way we stay compatible with older eventcore versions and keep the platform knowledge a bit more confined to one place.